backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -450,6 +472,175 @@ describe("Listings", () => {
     expect(listingsSearchResponse.body.items.length).toBe(1)
     expect(listingsSearchResponse.body.items[0].name).toBe(newListingName)
   })
+  describe("listings approval notification", () => {
+    let listing: ListingUpdateDto, adminId, alameda
+    beforeAll(async () => {
+      adminId = (await userRepository.find({ where: { email: "admin@example.com" } }))?.[0]?.id
+      alameda = (await jurisdictionsRepository.find({ where: { name: "Alameda" } }))[0]
+      const queryParams = {
+        limit: "all",
+        filter: [
+          {
+            $comparison: "=",
+            name: "Test: Draft",
+          },
+        ],
+      }
+      const query = qs.stringify(queryParams)
+      const res = await supertest(app.getHttpServer()).get(`/listings?${query}`).expect(200)
+      listing = { ...res.body.items[0] }
+    })
+    it("update status to pending approval and notify appropriate users", async () => {
+      listing.status = ListingStatus.pendingReview
+      const putPendingApprovalResponse = await supertest(app.getHttpServer())
+        .put(`/listings/${listing.id}`)
+        .send(listing)
+        .set(...setAuthorization(adminAccessToken))
+        .expect(200)
+
+      const listingPendingApprovalResponse = await supertest(app.getHttpServer())
+        .get(`/listings/${putPendingApprovalResponse.body.id}`)
+        .expect(200)
+
+      expect(listingPendingApprovalResponse.body.status).toBe(ListingStatus.pendingReview)
+      expect(mockRequestApproval).toBeCalledWith(
+        expect.objectContaining({
+          id: adminId,
+        }),
+        { id: listing.id, name: listing.name },
+        expect.arrayContaining(["admin@example.com", "mfauser@bloom.com"]),
+        "http://localhost:3001"
```

Comment:
I'd recommend not hardcoding this to the url and rather have it be `process.env.PARTNERS_PORTAL_URL`. Same with the other tests that have this. That way these tests will pass no matter what the url is set to in the environment it is in.